### PR TITLE
Add: search API docs

### PR DESCRIPTION
**Pull request by DaKingKong**

### CODE REVIEW

File: docs/basics/pagination.md [hunk 1]
```diff
@@ -36,14 +36,112 @@ API methods that supports pagination takes two (optional) parameters:
...
+
+Endpoint: [GET] `https://{account_name}.api.{platform_hostname}/1.0/content_threads`
+
+Auth: Bearer `{accessToken}` (refer to [Authentication](../basics/auth.md))
+
+Query parameter: `q={filters}` (eg. `q=assigned_to_me:"true"`)
+
+#### Parameters
+
+| parameter | value | description |  |
```
[the remark this answers is not in the document]
Comment (DaKingKong) [reply docs/basics/pagination.md]: Table format tool generated it by mistake. I've deleted it and pushed a new commit

File: docs/basics/pagination.md [hunk 2]
```diff
@@ -36,14 +36,112 @@ API methods that supports pagination takes two (optional) parameters:
...
+| needs_manual_categorization | true | Restrict threads having a content that needs manual categorization (meaning ICE could not classify a mandatory category) |
+| not_categorized_in |  | Restrict threads that do not contain one or several specific categories |  |
+| opened_and_assigned | true or false | Restrict on threads under open interventions. |  |
+| opened_and_assigned_to | user_id | Restrict on thread under open intervention assigned to specified user id. |  |
+| opened_and_assigned_to_disabled_agents | true | Restrict on threads under open interventions assigned to disabled agents. |  |
+| opened_and_assigned_to_me | true or false | Restrict on threads under your open interventions. |  |
+| opened_and_assigned_to_team | team_id | Restrict on threads under open interventions of given team id. Negation is supported. |  |
+| status_in | name | Restrict on thread with least one of specified status. Accepted statuses are new (a freshly created), assigned (under intervention), forwarded, replied (replied by an agent), user_reply (a reply made by an agent), user_initiated (initiated by an agent) or ignored. Negation can be specified using a minus before keyword. |
```
[the remark this answers is not in the document]
Comment (DaKingKong) [reply docs/basics/pagination.md:103]: same as above

File: docs/basics/pagination.md [hunk 3]
```diff
@@ -36,14 +36,112 @@ API methods that supports pagination takes two (optional) parameters:
...
+| categorized_in_mine | true or false | Restrict on your categories. It matches one of your categories. |
+| categorized_in_mine_of | category_id | Restrict threads containing categories in at least one of your categories in given category id. Negation is not supported. |
+| first_content_after | date | Restrict threads where their first content is created after given date. Negation is not supported. |
+| first_content_before | date | Restrict threads where their first content is created before given date. Negation is not supported. |
+| foreign | id | Restrict threads with the mentioned foreign_id. Negation is not supported. |
+| forwarded_into | category_id | Restrict threads containing forwarded contents in at least one of given category id. Negation is supported. |
+| forwarded_into_mine | true or false | Restrict threads containing forwarded contents in at least one of your categories. |
+| forwarded_into_mine_of | category_id | Restrict threads containing forwarded categories in at least one of your categories in given category id. Negation is not supported. |
+| language |  | Restrict threads by content language. |
+| language |  | Restrict threads by content language. |
```
Comment (DrissTM) [inline docs/basics/pagination.md:93]: duplicated row

```suggestion
| language |  | Restrict threads by content language. |
```

**Review by DrissTM — CHANGES_REQUESTED** [review]
one small suggestion, and LGTM after that 👍 

thanks for taking care of this BTW 🙏

**Pull request merged**